src/components/consul/detail/ConsulVotingComponent.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import { TitleContainer, Title, TitleShadow } from '../../Title';
+import { Wrapper, WrapperRow } from '../../Wrapper';
+import { texts, consts, device, colors, Icon, normalize } from '../../../config';
+import { RegularText } from '../../Text';
+import { Touchable } from '../../Touchable';
+
+const text = texts.consul;
+const a11yText = consts.a11yLabel;
+
+export const ConsulVotingComponent = (votesData) => {
+  const { cachedVotesTotal, cachedVotesUp, cachedVotesDown } = votesData.votesData;
+
+  let upVotesPercent = 0;
+  let downVotesPercent = 0;
+  if (cachedVotesTotal) {
+    upVotesPercent = (cachedVotesUp * 100) / cachedVotesTotal;
+    downVotesPercent = (cachedVotesDown * 100) / cachedVotesTotal;
+  }
+
+  const onVoting = (like) => {
+    //TODO: I need mutation
```

Comment:
do we find this in the poll PR?

Reply:
Sorry, I didn't understand exactly what you mean. 😔

Reply:
is this still a todo or is there updated code in the latest changes with the pull request for polls?

Reply:
The code has been updated. This todo has been deleted.